lib/Dialect/LWE/IR/LWEDialect.cpp

```diff
@@ -177,6 +177,18 @@ LogicalResult ReinterpretUnderlyingTypeOp::verify() {
   return success();
 }
 
+// Verification for RLWE_EncryptOp
+LogicalResult RLWEEncryptOp::verify() {
+  // Placeholder verification
```

Comment:
I think you will need to rebase over main to remove this spurious extra code, here and in LWEOps.td